.circleci/config.yml

```diff
@@ -87,7 +87,7 @@ jobs:
             .circleci/shared-helpers
       - *restore_npm_cache
       - node/install-npm:
-          version: "7"
+          version: '8'
```

Comment:
This should be `'^8'`. See https://financialtimes.slack.com/archives/C3TJ6KXEU/p1652356882216429

Reply:
Ooh I didn't understand exactly what people were saying in that thread. So if we don't have the `^` and they release a new version of npm under `8` then our builds will fail?